elements/pfe-timestamp/docs/pfe-timestamp.md

```diff
@@ -0,0 +1,181 @@
+{% renderOverview %}
+  A timestamp provides consistent formats for displaying date and time values.
+
+  Default:**<pfe-timestamp></pfe-timestamp>**
```

Comment:
is this meant to display at a code snippet or a live example? 
if code snippet:
~~~suggestion
  Default:
  ```html
  <pfe-timestamp></pfe-timestamp>**
  ```
~~~

if live example:
```suggestion
  Default: <pfe-timestamp></pfe-timestamp>
```

otherwise, I guess this will render as 
```html
<strong><pfe-timestamp></pfe-timestamp></strong>
```

it's a matter of taste i guess, but I like to use `<dl>` for things like this